packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -35,7 +35,7 @@ import * as icons from "../../asset/icon";
 
 export default {
   name: "TodoItem",
-  props: ["commands", "todo"],
```

Comment:
Not sure about this one. I think, I would still inject the commands as a dependency via props.